clickhouse-native-jdbc/src/main/java/com/github/housepower/jdbc/settings/SettingKey.java

```diff
@@ -146,7 +146,8 @@
     host(ISettingType.String, ""),
     database(ISettingType.String, ""),
     password(ISettingType.String, ""),
-    query_timeout(ISettingType.Seconds, "");
+    query_timeout(ISettingType.Seconds, ""),
+    characterEncoding(ISettingType.String, "charset for converting between Bytes and String");
```

Comment:
`characterEncoding` is MySQL style parameter, we can use `charset` to unify parameter style